tests/test_extractor.py

```diff
@@ -30,3 +33,125 @@ def test_multiple_chunks(self, tmp_path: Path):
         assert sorted(tmp_path.iterdir()) == [written_path1, written_path2]
         assert written_path1.read_bytes() == content[:4]
         assert written_path2.read_bytes() == content[4:]
+
+
+def test_fix_permissions():
+
+    with tempfile.TemporaryDirectory() as tempdir:
+        tmp_dir = Path(tempdir)
```

Comment:
There is a pytest fixture, which can help you achieve this without using the `tempfile` lib at all: https://docs.pytest.org/en/6.2.x/tmpdir.html

e.g. These two are the same:

```python
def test_fix_permissions():
    with tempfile.TemporaryDirectory() as tempdir:
        tmp_dir = Path(tempdir)
        test_dir = tmp_dir / "dir_a"
```

```python
def test_fix_permissions(tmp_path: Path):
    test_dir = tmp_path / "dir_a"
```